.travis.yml

```diff
@@ -30,87 +32,54 @@ matrix:
   include:
     - os: linux
       env:
-        - MB_PYTHON_VERSION=2.7
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=2.7
-        - UNICODE_WIDTH=16
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=2.7
-        - PLAT=i686
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=2.7
-        - PLAT=i686
-        - UNICODE_WIDTH=16
-    - os: linux
-      env:
-        - MB_PYTHON_VERSION=3.4
-        - NP_TEST_DEP=numpy==1.9.3
+        - MB_PYTHON_VERSION=3.5
     - os: linux
       env:
-        - MB_PYTHON_VERSION=3.4
+        - MB_PYTHON_VERSION=3.5
         - PLAT=i686
-        - NP_TEST_DEP=numpy==1.9.3
-    - os: linux
+    - os: osx
+      language: generic
       env:
         - MB_PYTHON_VERSION=3.5
-        - NP_BUILD_DEP=numpy==1.9.3
-    - os: linux
+    - os: osx
+      language: generic
       env:
         - MB_PYTHON_VERSION=3.5
-        - PLAT=i686
-        - NP_BUILD_DEP=numpy==1.9.3
+        - SCIPY_BUILD_DEP=scipy==0.17.1
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.6
-        - NP_BUILD_DEP=numpy==1.11.3
         - DAILY_BUILD=true
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.6
         - PLAT=i686
-        - NP_BUILD_DEP=numpy==1.11.3
         - DAILY_BUILD=true
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.7
-        - NP_BUILD_DEP=numpy==1.14.5
-        - NP_TEST_DEP=numpy==1.14.5
+        - NP_BUILD_DEP=numpy==1.16.1
+        - NP_TEST_DEP=numpy==1.16.1
         - DAILY_BUILD=true
     - os: linux
       env:
         - MB_PYTHON_VERSION=3.7
         - PLAT=i686
-        - NP_BUILD_DEP=numpy==1.14.5
-        - NP_TEST_DEP=numpy==1.14.5
+        - NP_BUILD_DEP=numpy==1.16.1
+        - NP_TEST_DEP=numpy==1.16.1
         - DAILY_BUILD=true
-    - os: osx
-      language: generic
-      env:
-        - MB_PYTHON_VERSION=2.7
-    - os: osx
-      language: generic
-      env:
-        - MB_PYTHON_VERSION=3.4
-    - os: osx
-      language: generic
-      env:
-        - MB_PYTHON_VERSION=3.5
-        - NP_BUILD_DEP=numpy==1.9.3
     - os: osx
       language: generic
       env:
         - MB_PYTHON_VERSION=3.6
-        - NP_BUILD_DEP=numpy==1.11.3
+        - NP_BUILD_DEP=numpy==1.16.1
         - DAILY_BUILD=true
     - os: osx
       language: generic
       env:
         - MB_PYTHON_VERSION=3.7
-        - NP_BUILD_DEP=numpy==1.14.5
-        - NP_TEST_DEP=numpy==1.14.5
+        - NP_BUILD_DEP=numpy==1.16.1
```

Comment:
Is this really necessary? We should try to build on the oldest supported version of numpy (oldest supported by us and compatible with Python 3.7) to make sure that the binary generated is compatible with those old numpy versions.

Testing on most recent numpy versions is fine and advised however.

Reply:
Not sure it's necessary. This is more for the daily build than release. Happy to leave NP_BUILD_DEP as the default (oldest) for all daily builds if that's what you're suggesting. Feel free to make the changes.

Reply:
Done.